apps/src/applab/designElements/dropdown.jsx

```diff
@@ -180,20 +181,122 @@ class DropdownEvents extends React.Component {
   }
 }
 
+const svgArrowUrl = color =>
```

Comment:
Would it make more sense to just use unicode arrows for this?

Reply:
Possibly? I was pretty happy we found a solution to this problem, but there may be other options.

Reply:
Yeah, you do have a pretty cool solution here. Since the norm in our repo is fa arrows, we should eventually move this to using fa. This shouldn't block the applab themes feature, but make a task in our backlog to change it later.